data/json/snippets.json

```diff
@@ -246,6 +246,166 @@
   },
   {
     "type": "snippet",
+    "category": "flier",
+    "text": [ "This is a flier for red ryder bbguns.  On it a child is pulling a shining red wagon with a cooked pheasant on it and a wooden rifle over one shoulder.  the child has a dog trailing beside him and a satisfied look on his face.  The caption read \"When you chose Red Ryder, you invest in the American Dream.  You invest in our Independence.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "flier",
+    "text": [ "This is an old flier for a movie from the 30s.  A tan man with slick black hair and muscles bulging through his offwhite suit is clasping a woman to his hip with one hand, and the woman is wearing a black leather dress.  With her hips splayed, she is holding a pistol in one hand and starring into directly out.   The caption reads \"Witness the rebirth of New Noir with 'Jersey Shore Blues'.  Starring Jenifer Languiz as 'Snookie'!\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "flier",
+    "text": [ "\"Have you found your savior today?  Atom is here for you, it is inside you!\"  This flier proves the rising popularity of the Atom Cult, a religion in which the essence of life is discovered through meticulous study and pious donations." ]
+  },
+  {
+    "type": "snippet",
+    "category": "flier",
+    "text": [ "\"Joe's Diner, 1/2 pound of meat, 3 toppings, \"your choice\", all with a side of freedom fries and a BIG Gulp size pop.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"WE WERE RIGHT THE GOVERNMENT DID IT\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"I shot the sheriff; but I couldn't find the deputy\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Some plant vines started chasin after me, so I took a gas mask and some teargas and I ran through them.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Slingshot right through the windshield k?\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"When I was a kid I used to slingshot at  bugs and birds.  Its really playing off nowadays, Ill tell you what\"" ]
```

Comment:
This could be kept as flavor text, since some people don't always format consistently.